lib/Elastica/Query/SpanContaining.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace Elastica\Query;
+
+/**
+ * SpanContaining query.
+ *
+ * @author Alessandro Chitolina <alekitto@gmail.com>
+ *
+ * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-span-containing-query.html
+ */
+class SpanContaining extends AbstractSpanQuery
+{
+    /**
+     * Constructs a SpanContaining query object.
+     *
+     * @param AbstractSpanQuery $little OPTIONAL
+     * @param AbstractSpanQuery $big OPTIONAL
+     */
+    public function __construct(AbstractSpanQuery $little = null, AbstractSpanQuery $big = null)
+    {
+        if (null !== $little) {
+            $this->setLittle($little);
+        }
+
+        if (null !== $big) {
+            $this->setBig($big);
+        }
+    }
+
+    /**
+     * @param AbstractSpanQuery $little
+     */
+    public function setLittle(AbstractSpanQuery $little)
+    {
+        $this->setParam('little', $little);
```

Comment:
We normal have a `return $this->setParam('little', $little);` in here so all setters return the object itself. This allows for chaining functions.